request/parser.go

```diff
@@ -84,3 +90,21 @@ func (p *Parser) getServiceID(host string) (relayer.ServiceID, error) {
 
 	return serviceID, nil
 }
+
+/* Parser Error Response */
```

Comment:
I think this should go into `errors.go` as well.

Reply:
Done.